var/spack/repos/builtin/packages/py-neuror/package.py

```diff
@@ -10,15 +10,22 @@
 class PyNeuror(PythonPackage):
     """Python morphology manipulation toolkit"""
 
-    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/nse/morph-repair"
-    git      = "ssh://bbpcode.epfl.ch/nse/morph-repair"
+    homepage = "https://github.com/BlueBrain/NeuroR"
+    git = "https://github.com/BlueBrain/NeuroR.git"
+    url = "https://pypi.io/packages/source/n/neuror/neuror-1.1.11.tar.gz"
 
     version('develop', branch='master')
-    version('1.0.0', tag='NeuroR-v1.0.0')
+    version('1.1.11', tag='NeuroR-v1.1.11')
 
     depends_on('py-setuptools', type=('build', 'run'))
 
-    depends_on('py-morph-tool', type='run')
-    depends_on('py-pandas', type='run')
-    depends_on('py-pyquaternion', type='run')
-    depends_on('py-plotly-helper', type='run')
+    depends_on('py-click@0.7.0:', type='run')
+    depends_on('py-matplotlib@2.2.3:', type='run')
+    depends_on('py-morph-tool@0.1.14:', type='run')
+    depends_on('py-morphio@2.1.1:', type='run')
+    depends_on('py-neurom@2.0.0.dev0:', type='run')
+    depends_on('py-nptyping@1.3.0:', type='run')
```

Comment:
@lguyot was the first one to use this package. I took it because it allows to specify the number or rows and columns of the np.array in the type hint but I don't know if it can do more.